### ISSUE TITLE
Reward snippet not showing and JS error being shown

### ISSUE THREAD

**Issue by chrismore** [mozilla/all-aboard#291]
The reward snippet is not being displayed and is consistent among both tracks.

It throws this message:

```
JPM [error]   Message: TypeError: SNIPPET_COPY[track] is undefined
  Stack:
    replaceSnippetCopy@resource://gre/modules/commonjs/toolkit/loader.js -> resource://all-aboard-dev/index.js:760:30
modifyAboutHome/aboutHome<.onAttach@resource://gre/modules/commonjs/toolkit/loader.js -> resource://all-aboard-dev/index.js:805:34
emitOnObject@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:112:9
emit@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:89:38
@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/page-mod.js:178:7
emitOnObject@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:112:9
emit@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:89:38
emitOnObject@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:123:45
emit@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:89:38
@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/content/worker.js:154:3
dispatch@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/method/core.js:119:12
@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/page-mod.js:189:3
emitOnObject@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:112:9
emit@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:89:38
emitOnObject@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:112:9
emit@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:89:38
emitOnObject@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:123:45
emit@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/event/core.js:89:38
messageReceived@resource://gre/modules/commonjs/toolkit/loader.js -> resource://gre/modules/commonjs/sdk/remote/parent.js:96:37
```

The issue is happening because this function call:

https://github.com/mozilla/all-aboard/blob/master/index.js#L805
var snippetContent = replaceSnippetCopy(track, contentURL, imageURL);

is looking in the array of SNIPPET_COPY (https://github.com/mozilla/all-aboard/blob/master/index.js#L50)

Going to probably have to move the snippet copy out of the html and put it in the array.

**Comment by chrismore** [comment]
@schalkneethling @FrancescoSTL found another bug and JS error

**Comment by chrismore** [comment]
This one should be easy too

**Comment by chrismore** [comment]
I got this.

**Comment by FrancescoSTL** [comment]
Fixed in #293, thanks @chrismore!
